src/vues/JeuVue.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace trivial\vues;
+
+use trivial\vues\GlobaleVue;
+
+class JeuVue {
+
+    public function renderPlateau($plateau){
```

Comment:
This is not OK. You can't have a templating engine AND generate html with string concatenation.